src/metabase/models/action.clj

```diff
@@ -250,7 +250,7 @@
   "Hydrates action from DashboardCard."
```

Comment:
does it still makes sense for this hydration to be under dashcard tho? should it be in dashboard ns instead?